Springer Link.js

```diff
@@ -196,13 +197,17 @@ function complementItem(doc, item) {
 			item.abstractNote = abs.trim();
 		}
 		item.abstractNote = titleTextGerman + "\n\n" + ZU.trimInternal(abs).replace(/^Abstract[:\s]*/, "");
-		if (item.abstractNote === null) item.abstractNote = '';
-	}
+    }
+    if (!item.abstractNote)
+        item.abstractNote = '';
 
-	let tags = ZU.xpathText(doc, '//span[@class="Keyword"] | //*[contains(concat( " ", @class, " " ), concat( " ", "c-article-subject-list__subject", " " ))]//span');
+	let tags = ZU.xpathText(doc, '//span[@class="Keyword"] | //*[contains(concat( " ", @class, " " ), concat( " ", "c-article-subject-list__subject", " " ))]//span | \
+               //li[@class="c-article-subject-list__subject"]');
 	if (tags && (!item.tags || item.tags.length === 0)) {
 		item.tags = tags.split(',');
-	}
+    }
+    // Trim and deduplicate
+    item.tags = [...new Set(item.tags.map(keyword => keyword.trim()))];
```

Comment:
never seen a spread operator `...` used like this before. Are you sure it works correctly?

Reply:
Yes, c.f. e.g https://stackoverflow.com/questions/20069828/how-to-convert-set-to-array